apps/website-new/docs/en/practice/monorepos/index.mdx

```diff
@@ -0,0 +1,99 @@
+
+### Shared Code Strategies
+
+Sharing code between polyrepos is difficult. It tends to involve creating and maintaining a private package registry of some form. Shared code gets built and deployed to this registry and consumed by the other polyrepos. If a change is made to the shared code, it must be manually released to the registry.
+
+This can be time-consuming and error-prone! It introduces the risk of blockers to feature work if any of the teams rely on the PR for the shared code to be released to the registry.
+
+It also circles back to the feedback loops issue, as teams must wait for the registry to be updated before they can see the impact of their changes, if they ever see the impact at all because of the isolated teams. They could be inadvertently introducing breaking changes that affect other teams.
+
+The teams consuming the shared code must also actively update the versions of the shared packages they are using to ensure compatibility across the module federated system, or risk breaking changes at runtime.
+
+### Dependency Management
+
+Polyrepos can also lead to dependency management issues. When teams are working in isolation in their own repository, they may have different dependencies or different versions of dependencies. This can lead to conflicts, versioning issues or worse, increased bundle sizes because of duplicated dependencies.
+
+For example, if one team is using React 18 and another team decides they want to use React 19 so they update their dependency, then once the changes are deployed to production there is a high risk of incompatible code and runtime errors, especially if other portions of the application do not support React 19.
+
+In a polyrepo setup, the update of React 19 does not immediately inform the team that by doing so they will be breaking the other teams. This can lead to a lot of frustration and rolling back of changes.
+
+### Testing Strategies
+
+When teams are working in isolation, they may not have the same testing strategies or tools. This can lead to inconsistent testing across the system, which can be a problem for the overall quality of the application.
+
+It also adds difficulty to the QA process. Either they test each micro frontend separately or they test all of them together.
+
+Testing individual micro frontends does not provide the same level of confidence in the overall quality of the application because it does not test the integration points nor any shared context between them.
+
+It also means having to build some kind of test framework that can mock certain layers of the full application which introduces a maintenance tax that must be paid to ensure that the test framework stays up to date with the other portions of the application.
+
+Testing the full application will provide a higher level of confidence overall, however, in a polyrepo setup it requires standing up configurable test environments that can deploy the latest versions of each micro frontend.
```

Comment:
Not sure what this means: `in a polyrepo setup it requires standing up configurable test environments `

is it `setting up` instead of `standing up`?

Reply:
yes should be setting up